baselines/driving_smarts/benchmark.py

```diff
@@ -0,0 +1,140 @@
+# smarts/benchmark/driving_smarts_competition_benchmark.py
+import argparse
+import logging
+from functools import partial
+
+import gymnasium as gym
+import ray
+
+from smarts.benchmark import auto_install
+from smarts.benchmark.driving_smarts import DEFAULT_CONFIG, load_config
```

Comment:
Could we just directly import `from smarts.core.utils.resources import load_yaml_config` instead of introducing `load_config`?

Reply:
I made this abstraction because I saw the potential need to modify the configuration during loading that would be beyond just loading a `.yaml`.